puppet/modules/quickstack/manifests/pacemaker/ceilometer.pp

```diff
@@ -52,6 +52,18 @@
       Anchor['redis-begin']
       Exec['redis-master-is-up'] ->
       Exec['i-am-ceilometer-vip-OR-ceilometer-is-up-on-vip']
+
+      quickstack::pacemaker::constraint::base {"redis-then-ceilometer-central-contraint":
```

Comment:
@cwolferh do you think maybe I should move this to constraints.pp?  I think that was meant to be more for cover inter-service constraints, wasn't it?